src/config/ConfigManager.cpp

```diff
@@ -1199,52 +1200,66 @@ void CConfigManager::handleSource(const std::string& command, const std::string&
         parseError = "source path " + rawpath + " bogus!";
         return;
     }
+    glob_t glob_buf;
+    memset(&glob_buf, 0, sizeof(glob_buf));
 
-    auto value = absolutePath(rawpath, configCurrentPath);
-
-    if (!std::filesystem::exists(value)) {
-        Debug::log(ERR, "source= file doesnt exist");
-        parseError = "source file " + value + " doesn't exist!";
+    if (glob(rawpath.c_str(), GLOB_TILDE, nullptr, &glob_buf) != 0) {
+        globfree(&glob_buf);
```

Comment:
instead of repeating this, we can make glob_buf a `std::unique_ptr` with a custom destructor

```cpp
std::unique_ptr<glob_t, void(*)(glob_t*)> globBuf(new glob_t, [](glob_t* g) { globfree(g); });
```

Reply:
Certainly, too archaic is mine.